test/export/test_export.py

```diff
@@ -2040,6 +2040,20 @@ def forward(self, x):
         ):
             export(Module(), (torch.tensor(1, device="cpu"),))
 
+    def test_float_conversion(self):
+        class Module(torch.nn.Module):
+            def forward(self, x):
+                return x.float()
```

Comment:
Can we add a test case that make sure the mutation on the output is banned?

Reply:
we already have one test doing this.